pkg/collector/metricdefine/define.go

```diff
@@ -0,0 +1,139 @@
+package metricdefine
```

Comment:
This is a type definition. Instead of use the `metricdefine` name, can we just use a `type` package? So we will have `pkg/collector/metric/type/type.go`

Reply:
em... we can't name a package as type.

Reply:
Sorry, it should be `types` in plural.

We already have this in our code and it is commonly used in other projects.
See here: https://github.com/sustainable-computing-io/kepler/blob/main/pkg/model/types/types.go

Reply:
updated